docs/api.md

```diff
@@ -111,11 +111,11 @@ The response will be a JSON object containing the following keys:
   [Interaction Pings](#interaction-pings), below. This field may be null, in
   which case no click ping is required for this suggestion provider.
 
-- `provider` - A string that identifies the source of this suggestion. This can
-  be used along with `block_id` to uniquely identify this suggestion. It is not
-  intended to be directly displayed to the user.
+- `provider` - A string that identifies the provider of this suggestion, such as
```

Comment:
If this is not meant to be displayed, why are we reporting it?

Reply:
We're including it in the API response because Firefox could use it in the future, for instance, performing different suggestion handling on different providers. Though that is largely irrelevant to the Firefox users, so they won't see it.